tests/PHPUnit/System/expected/test_DataComparisonTest_multipleAgainstLongMultiple__VisitFrequency.get_day.xml

```diff
@@ -26,9 +26,22 @@
 				<bounce_count_new>11</bounce_count_new>
 				<sum_visit_length_new>0</sum_visit_length_new>
 				<max_actions_new>1</max_actions_new>
+				<nb_profilable_new>0</nb_profilable_new>
 				<bounce_rate_new>100%</bounce_rate_new>
 				<nb_actions_per_visit_new>1</nb_actions_per_visit_new>
 				<avg_time_on_site_new>0</avg_time_on_site_new>
+				<nb_uniq_visitors_returning>0</nb_uniq_visitors_returning>
+				<nb_users_returning>0</nb_users_returning>
+				<nb_visits_returning>0</nb_visits_returning>
+				<nb_actions_returning>0</nb_actions_returning>
+				<nb_visits_converted_returning>0</nb_visits_converted_returning>
+				<bounce_count_returning>0</bounce_count_returning>
```

Comment:
Are you able to explain why there are so many test file updates, where a lot other metrics (than only the profilable metric) has been added?

Reply:
I haven't verified, but I'm guessing the returning VisitsSummary.get API method returned an empty DataTable (w/ no metrics instead of 0 values). Something must have changed here to return a DataTable w/ entries for 0 metric values. Is it important?

Reply:
I guess the reason might be, that for `nb_profilable` there is a special handling in the archive writer to also write empty archives. As the additional rows are result of an segment, I guess the segment doesn't have any visits and before no archives were written and such no values were available. Now the nb_profilable archive is written nevertheless and the api requests for that segment seems to return 0 values instead.
I think for the VisitFrequency result it's better to have 0 values, than not having the metrics included at all. But not sure if this might have other implications maybe.

Reply:
I'll see if I can make it behave as before.

Reply:
updated